config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -15,4 +17,57 @@ periodics:
     containers:
     - image: gcr.io/k8s-testimages/kubekins-e2e:v20190322-1960d2fdd-1.12
       command:
-      - "./scripts/ci-aws-cred-test.sh"
\ No newline at end of file
+      - "./scripts/ci-aws-cred-test.sh"
+      securityContext:
+        privileged: true
+- name: periodic-cluster-api-provider-aws-bazel-e2e
+  decorate: true
+  interval: 1h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-service-account: "true"
```

Comment:
I am un-sure if we need the preset aws-ssh and preset aws-credentials for this job. Please verify.